docs/identities/sign-in/check-session.mdx

```diff
@@ -0,0 +1,140 @@
+---
+id: check-session
+title: Check for user login sessions in browser cookies and session tokens
```

Comment:
```suggestion
title: Check for user session status in browser cookies and session tokens
```

we check the session status not the login status, right? 
So checking login status would be e.g. checking the status of the login flow, 
we check if the session has status "active" or not. 
At least that is how I would understand it.